### ISSUE TITLE
Loud sound before race?

### ISSUE THREAD

**Issue by MagitekArmor** [ksdomino/OR2LouderEngineSound#2]
Sound is balanced during gameplay, however before the race starts, during the countdown, sound is very loud. The car engine and the countdown itself.

Maybe I'm doing something wrong?

**Comment by ksdomino** [comment]
No you're probably doing everything right but I'll confirm with a test when I can. 
Most likely I overlooked the pre-race engine sound because my goal was to reduce all the sound effects APART from the engine sounds so it's most likely entirely my mistake.
It seems like I'll have to locate that sound and reduce it's volume. 
--I think it's part of the ICS.pak file-- so it shouldn't be too hard to find so I'll post an update when it's done. 
Update: It's not part of ICS.PAK so maybe that's why it was missed. I found those sounds are in PAS1.pak. I already have a lower volume version of that on mine so next I'll check it against the version I already uploaded and then upload my quieter one for you to try out.
